view/base.js

```diff
@@ -434,4 +437,4 @@ function clg( i ) {
 	if ( !window.console )return;
 	var l = arguments.length, o = 0;
 	while ( o < l )console.log(arguments[o++])
-}
\ No newline at end of file
+}
```

Comment:
This is a nitpick, but changes to spacing mess up git blame, it's too easy to avoid it here to not to :)